starknet_py/tests/e2e/docs/guide/test_serializing.py

```diff
@@ -0,0 +1,63 @@
+# pylint: disable=import-outside-toplevel, unused-variable
+import json
+
+from starknet_py.tests.e2e.fixtures.misc import read_contract
+
+
+def test_abi_parsing():
+    raw_abi_string = read_contract("erc20_abi.json")
+    # docs: start
+    from starknet_py.abi import AbiParser
+
+    # noinspection PyUnresolvedReferences,PyUnusedLocal
+    contract = """
+    @event
+    func Transfer(from_: felt, to: felt, value: Uint256) {
+    }
+    
+    @external
+    func transferFrom{syscall_ptr: felt*, pedersen_ptr: HashBuiltin*, range_check_ptr}(
+        sender: felt, recipient: felt, amount: felt
+    ) -> (success: felt) {
+        alloc_locals;
+        local success: felt = 1;
+        return (success,);
+    }
+    """
+
+    erc20_abi = json.loads(raw_abi_string)
+    abi = AbiParser(erc20_abi).parse()
```

Comment:
If `contract` is not used, let's just make it a multiline comment, removing `contract =` assignment. But ideally this example would show abi used by the rest of the methods.

Reply:
On the second though, abi is too big to be included in this example.